cnf-tests/testsuites/e2esuite/multinetworkpolicy/reachmatcher.go

```diff
@@ -0,0 +1,489 @@
+package multinetworkpolicy
+
+import (
+	"encoding/json"
+	"fmt"
+	"math/rand"
+	"net"
+	"strings"
+	"sync"
+
+	nadv1 "github.com/k8snetworkplumbingwg/network-attachment-definition-client/pkg/apis/k8s.cni.cncf.io/v1"
+	"github.com/onsi/gomega/format"
+	"github.com/onsi/gomega/types"
+	client "github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/client"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/images"
+	"github.com/openshift-kni/cnf-features-deploy/cnf-tests/testsuites/pkg/pods"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/utils/pointer"
+)
+
+const debugIPTablesImage = "quay.io/openshift/origin-multus-networkpolicy:4.12"
```

Comment:
this will not work for disconnected environments. I think is better to add the iptables package to our cnf-test image

Reply:
it happens cnf-tests already has iptables installed:

```
sh-5.1$ docker run quay.io/openshift-kni/cnf-tests:4.11 iptables -v
iptables v1.8.4 (nf_tables): no command specified
Try `iptables -h' or 'iptables --help' for more information.

sh-5.1$ docker run registry-proxy.engineering.redhat.com/rh-osbs/openshift4-cnf-tests:v4.11.1-4 iptables -v
iptables v1.8.4 (nf_tables): no command specified
Try `iptables -h' or 'iptables --help' for more information.
```

Going to add to to cnf-tests Dockerfile anyway.